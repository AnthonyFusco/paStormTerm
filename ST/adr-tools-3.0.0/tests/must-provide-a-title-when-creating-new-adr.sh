if adr new
then
    echo ERROR: should have failed
fi
