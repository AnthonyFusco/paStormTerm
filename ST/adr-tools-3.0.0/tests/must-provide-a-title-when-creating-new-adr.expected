if adr new
then
    echo ERROR: should have failed
fi
ERROR: no title given
